### ISSUE TITLE
Documentation: improve page on 'how filters work'

### ISSUE THREAD

**Issue by jedio60** [duplicati/duplicati#4134]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Description
I'd like to add something rather important to the wiki page [How do filters work?](https://github.com/duplicati/duplicati/wiki/How-do-filters-work%3F) but I don't have the knowledge yet. If this issues database is not the place for writing about the documentation, please let me know.

The page is not mentioning that (apparently) every specification of a specific file or folder name must **include the absolute path** of the file and that **`*/`** (`*\` on Windows) means 'in any folder'). I propose to explain this in the section 'Syntax' in the page. If it's already explained elsewhere in the wiki, maybe add a reference to that instead.

I found out the hard way, because an exclude filter `parent.lock` didn't work (I wanted this because of warnings about these locking files in Thunderbird profile folders). Changing this to  `*\parent.lock` (on Windows) solved the problem.

As a developer of course I understand that this mechanism is logical. It just seems good for all Duplicati users to describe it explicitly.

**Comment by KoalaBear84** [comment]
Yes. Sorry, I have to say this, filters in Duplicati 'suck'.. I've tried 3 options to exclude folders, none of them seem to work..

"Exclude directories whose names contain", "Exclude folder" and "Exclude file" (this one will be used when you uncheck a folder in the GUI.

And still.. It'll process and upload it? So, this ticket is very important.

![image](https://user-images.githubusercontent.com/5886900/82032224-361a1e00-969b-11ea-8fc0-05505a4cdaa4.png)
![image](https://user-images.githubusercontent.com/5886900/82032244-3c0fff00-969b-11ea-9fbf-c33294d8c4d0.png)
It is also uploading it's own sqlite databases every time, which are 1.86 GB and 724 MB currently..
![image](https://user-images.githubusercontent.com/5886900/82032257-3f0aef80-969b-11ea-954c-8d61ecb0584a.png)

**Comment by drwtsn32x** [comment]
@KoalaBear84 What version are you using?  In 2.0.5.1 and some prior versions, the filter builder in the UI is buggy and adds extra slashes.  It has been improved in the latest canary versions, but still not perfect.

I recommend using the "Edit as text" option instead of the dropdowns to do your filters for now.  Just don't switch back to "Edit as list" when you're done or the UI can foul up the filters again.

**Comment by drwtsn32x** [comment]
Here's a recent pull request that helped fix some of the issues:

https://github.com/duplicati/duplicati/pull/4112

**Comment by KoalaBear84** [comment]
@drwtsn32x Sorry, I didn't meant to be rude in any way. Yes, latest / 2.0.5.1.

Strange, when I use Edit as text everything is OK?

![image](https://user-images.githubusercontent.com/5886900/82114782-0c2a2f80-975f-11ea-9c28-429d3bcfda40.png)

Only when I use "Exclude folder" it will be like "C:\Windows\\\", but then "Exclude file" should work already, but it doesn't. It should exclude all files and directories in "C:\Windows\"?
